### PR TITLE
List of the with 1.3 features added

### PR DESCRIPTION
**Pull request by idvoretskyi**
Features for 1.3 list added.

### CODE REVIEW

File: release-1.3/features-1.3.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+|  **Item** | Node Out-of-memory handling |
+|  ------ | ------ |
+|  **Stage** | Stable |
+|  **Status** | Merged |
+|  **SIG** | Node |
+|  **Issue/PR Links** |  |
+|  **Dev Lead** | dchen1107  |
+|  **Assignee** | derekwaynecarr |
+|  **Entity** | Google / Red Hat |
+|  **Docs** | In Progress |
```
Comment (derekwaynecarr) [inline release-1.3/features-1.3.md]: docs are here:

https://github.com/kubernetes/kubernetes.github.io/pull/718

**Comment by pwittrock** [comment]
@idvoretskyi Is this the correct place for this?  Is the process documented somewhere?  I thought this was only for creating issues?

**Comment by idvoretskyi** [comment]
@pwittrock yes, it is - we have decided to use this repo for storing this information as well and agreed on the last two weekly community meetings.

**Comment by pwittrock** [comment]
SG.  Plz address to with Derek's comment and squash commits.  I'd like to get the actual docs links in there as a follow up.

**Comment by idvoretskyi** [comment]
@pwittrock done.

@derekwaynecarr thank you, updated.

**Pull request merged**